pages/science/reactions.rst

```diff
@@ -285,6 +285,47 @@ Chebyshev reactions can be defined in the CTI format using the
 :cti:class:`chebyshev_reaction` entry, or in the YAML format using the
 :ref:`Chebyshev <sec-yaml-Chebyshev>` reaction ``type``.
 
+.. _sec-Blowers-Masel:
+
+Blowers-Masel Reactions
+-----------------------
+
+Blowers-Masel rate expression represents the approximation porposed by Blowers
+and Masel [#BlowersMasel2000]_ to automatically scale activation energy
+as we change species' enthalpies. The activation energy estimation can be written
+as:
+
+.. math::
+
+   E_a = 0 \;\text{if } \Delta H \leq -4 E_a^0
+
+   E_a = \Delta H \;\text{if } \Delta H \geq 4 E_a^0
+
+   E_a =  \frac{\left( w + \frac{\Delta H }{2} \right)  (V_P - 2 w + \Delta H) ^2}
+               {V_P^2 - 4 w^2 + \Delta H^2} \;\text{Otherwise}
```

Comment:
```suggestion
.. math::

   E_a = \begin{cases}
      0 & \text{if } \Delta H \leq -4 E_a^0 \\
      \Delta H & \text{if } \Delta H \geq 4 E_a^0 \\
      \frac{\left( w + \frac{\Delta H }{2} \right)  (V_P - 2 w + \Delta H) ^2}
               {V_P^2 - 4 w^2 + \Delta H^2} & \text{Otherwise}
      \end{cases}
```

Reply:
Please also go into `conf.py` and delete the `filters.typogrify` filter from the list of applied filters. It isn't doing anything useful and just causes problems for us.